doc/source/spectra.rst

```diff
@@ -80,6 +131,98 @@ spectra:
   spec1d_col_1_5 = spec1d_col[1:5]
   fig2 = plot_1d(spec1d_col_1_5)
 
+Broadening
+----------
+
+A collection of 1D spectra can also be broadened using
+:py:meth:`Spectrum1DCollection.broaden <euphonic.spectra.Spectrum1DCollection.broaden>`,
+which broadens each spectrum individually, giving the same result
+as using :py:meth:`Spectrum1D.broaden <euphonic.spectra.Spectrum1D.broaden>`
+on each contained spectrum.
+
+Grouping By Metadata
+---------------------
+
+You can group and sum specific spectra from a ``Spectrum1DCollection``
+based on their metadata using
+:py:meth:`Spectrum1DCollection.group_by <euphonic.spectra.Spectrum1DCollection.group_by>`.
+For example, if you have a collection containing
+8 spectra with the following metadata:
+
+.. code-block:: py
+
+  {'line_data': [
+     {'index': 1, 'species': 'Si', 'weighting': 'coherent'},
+     {'index': 2, 'species': 'Si', 'weighting': 'coherent'},
+     {'index': 3, 'species': 'O', 'weighting': 'coherent'},
+     {'index': 4, 'species': 'O', 'weighting': 'coherent'},
+     {'index': 1, 'species': 'Si', 'weighting': 'incoherent'},
+     {'index': 2, 'species': 'Si', 'weighting': 'incoherent'},
+     {'index': 3, 'species': 'O', 'weighting': 'incoherent'},
+     {'index': 4, 'species': 'O', 'weighting': 'incoherent'}]
+  }
+
+If you want to group and sum spectra that have the same ``weighting``:
+
+.. code-block:: py
+
+  weighting_pdos = spec1d_col.group_by('weighting')
+
```

Comment:
I think it would be very helpful to include an output example of the metadata in the resulting pdos. (Or plot it?)

Reply:
Good point, I've added example output